proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -415,7 +415,7 @@ message ConsPubKeyRotationHistory {
   option (gogoproto.goproto_getters) = false;
 
   // operator_address defines the address of the validator's operator; bech encoded in JSON.
-  string operator_address = 1;
+  bytes operator_address = 1;
```

Comment:
Is there a reason we're choosing to ignore buf break output? It's present in CI to prevent buf breaking changes like this.